app/views/users/_user.haml

```diff
@@ -1,11 +1,8 @@
-
+- partial = 'users/follow'
 %tr
   %th= link_to user.name, user_path(user), :class => 'user-links'
```

Comment:
you can use `user` here instead of `user_path(user)`, `link_to user.name, user, ...`